src/new-client/public/manifest.json

```diff
@@ -4,48 +4,48 @@
   "description": "Real-time FX trading showcase by Adaptive.",
   "icons": [
     {
-      "src": "/static/media/reactive-trader.ico",
```

Comment:
as discussed offline, this is not the desired end state .. the deployment was supposed to be set up to strip out the ongoing branch/blah path for the 'env' deployments .. but that can be looked at in the meantime, as other comments, I cannot find this config anywhere, although I am sure Gary pointed it out somewhere ...